mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -529,7 +529,7 @@ class LinearRegressionSummary private[regression] (
     val predictionCol: String,
     val labelCol: String,
     val model: LinearRegressionModel,
-    val diagInvAtWA: Array[Double]) extends Serializable {
+    private val diagInvAtWA: Array[Double]) extends Serializable {
```

Comment:
This could be useful to users, but it should be documented if public.  @yanboliang Did you and @mengxr intend this to be public?